README.md

```diff
@@ -67,6 +67,7 @@ The creators of the below listed watchfaces are happy to answer your questions a
 | ![thumbnail](.thumbnails/karlos-matrix-round.jpg) |![thumbnail](.thumbnails/karlos-matrix.jpg) | [karlos-matrix](karlos-matrix/usr/share/asteroid-launcher/watchfaces/karlos-matrix.qml) | [eLtMosen](https://github.com/eLtMosen) |
 | ![thumbnail](.thumbnails/logo-45degree-round.jpg) |![thumbnail](.thumbnails/logo-45degree.jpg) | [logo-45degree](logo-45degree/usr/share/asteroid-launcher/watchfaces/logo-45degree.qml) | [eLtMosen](https://github.com/eLtMosen) |
 | ![thumbnail](.thumbnails/motivational-quotes-round.jpg) |![thumbnail](.thumbnails/motivational-quotes.jpg) | [motivational-quotes](motivational-quotes/usr/share/asteroid-launcher/watchfaces/motivational-quotes.qml) | [eLtMosen](https://github.com/eLtMosen) |
+| ![thumbnail](.thumbnails/numerals-duo-neon-green-round.jpg) |![thumbnail](.thumbnails/numerals-duo-neon-green.jpg) | [orbiting-asteroids](numerals-duo-neon-green/usr/share/asteroid-launcher/watchfaces/numerals-duo-neon-green.qml) | [MagneFire](https://github.com/MagneFire) |
```

Comment:
Nice, that looks sleek af :D
Here is an [Orbiting Asteroids] string in the description that was likely not edited after copy+paste.

Reply:
Thanks! Great catch too!